enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -22,10 +22,10 @@ import {
 } from "embedding-sdk/components/private/InteractiveQuestionResult";
 import { withPublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
 import type { SdkPluginsConfig } from "embedding-sdk/lib/plugins";
-import type { CardId } from "metabase-types/api";
+import type { Card } from "metabase-types/api";
 
 export type InteractiveQuestionProps = PropsWithChildren<{
-  questionId?: CardId;
+  questionId?: Card["id"];
```

Comment:
I think `CardId` should point to the same as `Card["id"]`, from `frontend/src/metabase-types/api/card.ts`:

```ts
export interface Card<Q extends DatasetQuery = DatasetQuery>
  extends UnsavedCard<Q> {
  id: CardId;
```

Reply:
yep I get it